### PR TITLE
update tests & PO to reflect report name change. Add xfails.

### PR DESCRIPTION
**Pull request by m8ttyB**
Remedy for issue #346 
* Updated POs
* Updated test
* Added xfail for prod
* Tweaked xfail for existing test failure.

### CODE REVIEW

File: pages/crash_stats_per_day.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 from pages.base_page import CrashStatsBasePage
 
 
-class CrashStatsPerActiveDailyUser(CrashStatsBasePage):
+class CrashStatsPerDay(CrashStatsBasePage):
 
     _page_title = 'Crashes per Active Daily User for Firefox'
```
Comment (bobsilverberg) [inline pages/crash_stats_per_day.py]: Is this title still accurate now that the report name has changed? If so, then maybe that's a bug that needs to be filed? I would have expected the title to change along with the report name.

**Comment by bobsilverberg** [comment]
It looks like I'm going to have to wait until tomorrow to test this, so I will complete my review then. Thanks @m8ttyB!

**Comment by bobsilverberg** [comment]
This looks good @m8ttyB. The tests now pass for me on stage and xfail on prod. It might be worth opening an issue to unxfail them when the changes make it to prod, as otherwise that is often forgotten. In fact, when running the tests there are already a number of XPasses that really should be looked at.

Also, what did you decide about the title? The expected title in the code is no longer accurate, I believe.

Finally, you have an extra merge commit in this PR because you pulled in a change (from my last PR). That should be removed prior to merging this PR so we don't have that extra merge commit in the repo.

Thanks!

**Comment by m8ttyB** [comment]
Thanks @bobsilverberg. It looks like this feature is shipping to prod in .. 3 .. 2 .. 1 now. I'll update the pr and remove the xfail for prod.

As for the other clean up .. xpasses and the page title thingy. I'll submit a seperate pr for those. The xpasses are interesting. I want to understand what under the hood changes the team has been making (last 24 - 80+ hours).  Sound kosher?

**Comment by bobsilverberg** [comment]
@m8ttyB I agree that the Xpasses should be treated as separate issues. I just mentioned them as an example of what can happen when we xfail something and then forget about it.

I still think we should at least update the title in that page object in *this* PR. I don't think it's a good idea to commit code that we know is incorrect, and the change to that page title does belong in the same PR where we are changing the name of the report in other places.

**Comment by m8ttyB** [comment]
The problem that to update the page title, we need to take into account the product (Firefox, B2G, WebAppRuntime, etc). I looked at the page titles on crash-stats.allizom.org and they properly take into account which product is under test. This tells me that this line of code is not being used. I suspect it's an artifact of some boiler plate PO design and that the code was never actually used or was used in a very limited and non-robust manner.

I'm 99.7% sure this code isn't used and can be removed. How about we split this; I'll file a git issue so the meta problem doesn't get lost.

**Comment by bobsilverberg** [comment]
@m8ttyB I'd still prefer that it just be addressed in this PR, but fine, let's make sure an issue gets filed about the inaccurate page title (which you are probably correct can just be removed) and then we can go ahead and merge this. There's still an issue with conflicts though. What I suggest is creating a new local branch off of the most recent `master` and then just cherry-pick the one commit from this into that.

**Comment by m8ttyB** [comment]
A bit of clean up and a new pr :-) https://github.com/mozilla/Socorro-Tests/pull/348

**Pull request closed without merging**